lib/get_player_input.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'tic_tac_toe.rb'
+
+def get_player_input(input, player, board=Array.new(3){Array.new(3, "_")})
+    puts "Player #{player}, enter your placeholder in format num1, num2"
+    input = input.gets.chomp
+
+    row = input.split(",")[0].to_i
+    col = input.split(",")[1].to_i
+
```

Comment:
As a refactor, you could also do this, to reduce the `split` duplication

```suggestion
    row, col = input.split(',').map(&:to_i)
```

Reply:
I like the proposed change - it looks nice and concise.

It's ok if it's hard to understand at first.  In that case, I recommend working through the code - left to right - in a systematic way:

1. `input` is a string like `"2,3"`.  Calling `split` returns an array like `["2", "3"]`
2. `.map` transforms/translates each element in the array, so that `"2"` becomes the number `2` and so on.  It returns an array like `[2, 3]`.
3. The way `map` works is by calling whatever function you pass to it.  The new syntax with `&:to_i` results in calling the `to_i` method on each `string` in the array.